controllers/mongodb_status_options.go

```diff
@@ -22,7 +22,7 @@ const (
 	None  severity = "NONE"
 )
 
-// optionBuilder is in charge of constructing a slice of options that
+// OptionBuilder is in charge of constructing a slice of options that
```

Comment:
this should match the function signature -- so should be `optionBuilder` -- let's revert this change.